ts/features/authentication/activeSessionLogin/store/reducer/index.ts

```diff
@@ -83,14 +121,38 @@ export const activeSessionLoginReducer = (
       };
 
     case getType(setFinishedActiveSessionLoginFlow):
+      return {
+        isActiveSessionLogin: false,
+        isUserLoggedIn: false,
+        activeSessionLoginLocalFlag: state.activeSessionLoginLocalFlag,
+        engagement: { ...state.engagement },
+        refreshMessagesSection: state.refreshMessagesSection
+      };
     case getType(consolidateActiveSessionLoginData):
     case getType(setLoggedOutUserWithDifferentCF):
     case getType(sessionCorrupted):
-      return activeSessionLoginIntialState;
-
+      return activeSessionLoginInitialState;
     default:
       return state;
   }
 };
 
-export const testable = isTestEnv ? activeSessionLoginIntialState : undefined;
+const CURRENT_REDUX_OPT_IN_STORE_VERSION = -1;
```

Comment:
```suggestion
const CURRENT_REDUX_ACTIVE_SESSION_STATE_STORE_VERSION = -1;
```
I guess this is a typo. Am I right?